psdk_wrapper/src/modules/camera.cpp

```diff
@@ -1216,8 +1352,65 @@ PSDKWrapper::camera_delete_file_by_index_cb(
   {
     RCLCPP_INFO(get_logger(), "Deleted file by index successfully.");
     response->success = true;
+  }
+
+  // Release rights
+  dji_release_downloader_rights(index);
+}
+
+void
+PSDKWrapper::camera_select_files_by_time_cb(
+    const std::shared_ptr<CameraSelectFilesByTime::Request> request,
+    const std::shared_ptr<CameraSelectFilesByTime::Response> response)
```

Comment:
if they do not explicitly have this natively, it has to go outside, since we are providing a wrapper we should not include extra functionality than the one given by Payload-SDK